ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -19,22 +20,29 @@ def __init__(self, driver):
         ocp_version = get_ocp_version()
         self.pvc_loc = locators[ocp_version]["pvc"]
 
-    def create_pvc_ui(self, sc_type, pvc_name, access_mode, pvc_size, vol_mode):
+    def create_pvc_ui(
+        self, project_name, sc_type, pvc_name, access_mode, pvc_size, vol_mode
+    ):
         """
         Create PVC via UI.
 
-        sc_type (str): storage class type
-        pvc_name (str): the name of pvc
-        access_mode (str): access mode
-        pvc_size (str): the size of pvc (GB)
-        vol_mode (str): volume mode type
+        Args:
+            project_name (str): name of test project
+            sc_type (str): storage class type
+            pvc_name (str): the name of pvc
+            access_mode (str): access mode
+            pvc_size (str): the size of pvc (GB)
+            vol_mode (str): volume mode type
 
         """
         self.navigate_persistentvolumeclaims_page()
 
-        logger.info("Select openshift-storage project")
+        logger.info(f"Search test project {project_name}")
         self.do_click(self.pvc_loc["pvc_project_selector"])
-        self.do_click(self.pvc_loc["select_openshift-storage_project"])
+        self.do_send_keys(self.pvc_loc["search-project"], text=project_name)
+
+        logger.info(f"Select test project {project_name}")
+        self.do_click((f"//a[normalize-space()='{project_name}']", By.XPATH))
```

Comment:
Not sure if we can access project_name in views.py file, because it is dynamic and name of the project changes every time, I will then have to create an object of TestPvcUserInterface class of test_pvc_ui file inside views.py file. Please suggest a suitable way to do so.

Reply:
@OdedViner Done